docs/types.rst

```diff
@@ -169,9 +169,13 @@ Fixed Point Numbers
 Rational and Integer Literals
 -----------------------------
 
-All number literals retain arbitrary precision until they are converted to a non-literal type (i.e. by
-using them together with a non-literal type). This means that computations do not overflow but also
-divisions do not truncate.
+Integer literals and rational number literals belong to a special type.
+The number literal type contains not just single literals
```

Comment:
Ok, but actually we have one type for every possible value for the literal, not for every possible expression.